exporter/loadbalancingexporter/trace_exporter.go

```diff
@@ -117,15 +128,11 @@ func (e *traceExporterImp) ConsumeTraces(ctx context.Context, td ptrace.Traces)
 		duration := time.Since(start)
 
 		if err == nil {
-			_ = stats.RecordWithTags(
-				ctx,
-				[]tag.Mutator{tag.Upsert(endpointTagKey, endpoints[exp]), successTrueMutator},
-				mBackendLatency.M(duration.Milliseconds()))
+			e.telemetry.LoadbalancerBackendLatency.Record(ctx, duration.Milliseconds(), metric.WithAttributeSet(exp.endpointAttr))
```

Comment:
Same as logs and metrics, I believe we can move the record latency call outside of the if/else statement.